xml/cl.xml

```diff
@@ -6326,6 +6327,7 @@ server's OpenCL/api-docs repository.
             </require>
             <require comment="cl_mem_properties_intel">
                 <enum name="CL_MEM_ALLOC_FLAGS_INTEL"/>
+                <enum name="CL_MEM_ALLOC_BUFFER_LOCATION_INTEL"/>
```

Comment:
This enum needs to be defined in its own block specifically for this new extension vs. adding it ot the USM block.  It's very possible that an implementation will support USM but not this enum.

The XML block specifically for this new extension could look something like:

```xml
        <extension name="cl_mem_alloc_buffer_location_intel" supported="opencl">
            <require>
                <type name="CL/cl.h"/>
            </require>
            <require comment="cl_mem_properties_intel">
                <enum name="CL_MEM_ALLOC_BUFFER_LOCATION_INTEL"/>
            </require>
            <require comment="cl_mem_alloc_info_intel">
                <enum name="CL_MEM_ALLOC_BUFFER_LOCATION_INTEL"/>
            </require>
        </extension>
```

Debatably there should also be sections for `cl_mem_properties` and `cl_mem_info` but this won't matter in practice.

Reply:
Thank you! I have moved buffer location reference in USM to extension's own definition (both in xml and asciidoc). Specifically:

Moved the definition in xml from `cl_mem_properties_intel` and `cl_mem_alloc_info_intel` to self contained `<extension name="cl_mem_alloc_buffer_location_intel"...`

Also removed extention's reference in `cl_intel_unified_shared_memory.asciidoc` and moved table buffer location row of `cl_mem_info_intel` to `extensions/cl_intel_mem_alloc_buffer_location.asciidoc`

Reply:
Thanks, the XML updates look good now.